make/e2e-setup.mk

```diff
@@ -269,6 +269,7 @@ e2e-setup-ingressnginx: $(call image-tar,ingressnginx) load-$(call image-tar,ing
 		--namespace ingress-nginx \
 		--create-namespace \
 		--set controller.image.tag=$(TAG) \
+		--set controller.image.registry=registry.k8s.io \
```

Comment:
We should bump the image and chart version instead of this parameter for current master, but this is ok for now as we also need to backport this so the e2e tests for supported released versions still work.